dotnet/targets/WorkloadManifest.windows.template.json

```diff
@@ -0,0 +1,32 @@
+{
+	"version": 5,
+	"workloads": {
```

Comment:
The reasoning behind the two files is that `WorkloadManifest.template.json` is used for all other platforms (mac, watch, tv), and these workloads don't (and some will never?) have a `*.Windows.Sdk` entry.

Rather than adding more complex string manipulation to `dotnet/Makefile` to conditionally add this windows entry to _only_ the iOS manifest, I added a new template for workloads that should include a "Windows" Sdk.

We may want add the `"platforms": [ "win-x86", "win-x64" ]` declaration here, though the conditional import in the iOS `WorkloadManifest.targets` file allows the new manifest which declares a windows specific pack to work on macOS when the pack is not installed.

Reply:
Ok makes sense 👍 

I think we need to put `platforms` so that the future `dotnet workload install` commands won't install these Windows packs on Mac.

Reply:
It looks like this feature may not be working yet, I'm going to revert this portion for now https://github.com/dotnet/sdk/issues/17151